srcpkgs/python-fonttools/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-fonttools'
+pkgname=python-fonttools
+version=3.43.1
+revision=1
+build_style=python-module
+pycompile_module="fonttools"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="${hostmakedepends}"
+short_desc="Library to manipulate font files from Python2"
+maintainer="svenper <svenper@tuta.io>"
+license="MIT, OFL-1.1, BSD-3-Clause"
+homepage="https://github.com/fonttools/fonttools"
+distfiles="https://github.com/fonttools/fonttools/archive/${version}.tar.gz"
+checksum=12b05c56ac60fe15b2fce6958df1408e054a403270ddfc05d3a096afe73d9419
+
+post_extract() {
+	mv ${XBPS_BUILDDIR}/fonttools-${version} ${wrksrc}
```

Comment:
Easier and the commonly used solution is to set `wrksrc="${pkgname#python-}-${version}"`

Reply:
Done!